src/editor/components/styles/controls.scss

```diff
@@ -31,9 +31,28 @@
   }
 }
 
+.extras-dropdown {
+  align-self: flex-end;
+  margin-left: auto;
+  // this pushes it away from the scroll bar, which looks nicer
+  margin-right: spacing(3);
+
+  & > button {
+    background: none;
+    font-family: "Headings Font";
+    border: none;
+
+    &:active,
+    &:hover {
+      background: none !important;
+    }
+  }
+}
+
 .control-popover {
   padding: $spacer;
   background: white;
-  border: 1px solid black;
+  border: solid 1px gray;
   border-radius: 5px;
```

Comment:
Should use one of our standardized units

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/68/commits/2a2cfeeea2d65a5ecf4a4b0c011de9c1175138c9